### PR TITLE
Ilariae/intro update

### PR DESCRIPTION
**Pull request by ilariae**
### Description

for the wormhole introduction page: minor fixes from feedback, added missing links and added supported networks at the end of the page
for the other pages: added missing links around learn pages, fixed links for the fundamentals pages that were broken due to folder reorg

### Checklist

- [x] **Required** - I have added a label to this PR 🏷️
- [x] **Required** - I have run my changes through Grammarly
- [ ] If pages have been moved, I have created redirects in the `wormhole-mkdocs` repo

### CODE REVIEW

**Review by albertov19 — APPROVED** [review]
LGTM

**Review by dawnkelly09 — DISMISSED** [review]
lgtm, thanks

File: learn/fundamentals/introduction.md [hunk 1]
```diff
@@ -89,4 +89,354 @@ More demos are available in the [demos page](#){target=\_blank}. <!-- demos page
 
 Wormhole supports a growing number of blockchains.
 
-<!-- List of Blockchains here -->
\ No newline at end of file
+<table data-view="cards" data-full-width="false">
```
Comment (eshaben) [inline learn/fundamentals/introduction.md:92]: This is copied from the other file right? Can a snippet be used instead so we only have to update the list in one place?

**Review by eshaben — COMMENTED** [review]
Nice! Thanks! Just one comment

File: learn/fundamentals/introduction.md (hunk 1 above)
Comment (ilariae) [reply learn/fundamentals/introduction.md:92]: yes i didnt find a snippet so I just copied it in, we should make it a snippet

**Review by eshaben — APPROVED** [review]
LGTM

**Pull request merged**